js/bibox.js

```diff
@@ -813,6 +813,14 @@ module.exports = class bibox extends Exchange {
         //         'status': 3
         //     }
         //
+        //
+        // withdraw
```

Comment:
The structure that is passed from `withdraw` to `parseTransaction` looks different from the comment below.